inc/views/template_parts.php

```diff
@@ -107,6 +108,15 @@ public function render_featured_post() {
 					'post_status' => 'publish',
 				)
 			);
+
+			/**
+			 * Filters the featured posts.
+			 *
+			 * @since 3.5.0
+			 *
```

Comment:
We should be sure the `since` announcement of the correct version (patch or minor according to the upcoming release)